tests/integration/setup-scripts/system_containers_setup.sh

```diff
@@ -43,3 +43,7 @@ export PYTHON=${PYTHON:-/usr/bin/python}
 export ATOMIC_OSTREE_REPO=${WORK_DIR}/repo
 export ATOMIC_OSTREE_CHECKOUT_PATH=${WORK_DIR}/checkout
 export NAME="test-system-container-$$"
+
+# This is to prevent the case where the ostree checkout path
+# can be non-existent when no container installation happens prior
+mkdir -p $ATOMIC_OSTREE_CHECKOUT_PATH
```

Comment:
why do we need to do this?

Reply:
ah, I noticed that the checkout path was never created until someone does a container installation. 
However, we will fail if the checkout path is not there for some actions( let's say we do `atomic images prune` ). 
```
# Check for dangling checkouts that couldn't be deleted on uninstall.
        checkouts = self._get_system_checkout_path()
        for i in os.listdir(checkouts):
```

It will error out for the above code when the checkout path is not existent. Therefore, to avoid such cases, we first make sure the checkout path exist. I caught the error when writing an integration test, and added it.

Reply:
but in our tests we always install a container before we try a prune right?  Does the issue happen in our tests suite without this change?

Reply:
Hmm, I added a new test, and in that test I tried to clean up the image each time I pull the image to ostree backend. Then the error happens if I did a `atomic images prune` without the checkout folder. To avoid it, i added the above line. Is that the correct way to do so?

Reply:
I think in general `atomic images prune` should not give an error if the checkout directory doesn't exist, but this is out of the scope of this PR.

If the tests pass without creating the checkout manually, let's drop it

Reply:
> If the tests pass without creating the checkout manually, let's drop it

ah, sorry for explaining it earlier, the test I add( system_containers_pull.sh) will fail if the checkout was not created. It will fail when the test is run by itself via (make test TEST_INTEGRATION=system_containers_pull). So, let's keep it in this PR for now, and I can address the above issue you mentioned in a separate PR?  WDYT